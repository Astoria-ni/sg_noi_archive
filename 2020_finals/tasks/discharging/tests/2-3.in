1476
131735 332811 1714211 2186312 2376271 3285650 3546293 3704477 5845743 7355886 7668159 7698553 8402316 8407034 9797402 11620170 13368088 15177332 15325594 15358209 16693926 17114955 17863149 17944370 17949699 18044615 18990736 19411667 19595793 19797846 22541443 23130624 24202434 25855484 26550742 26848678 26890555 28356424 28643838 29655365 29709387 31177581 31559763 31708699 31984545 32365054 35312617 36482626 37383935 38044975 39395351 39634190 40078102 40256615 40689785 40925184 40946751 41796950 42340988 45357099 47407618 47911648 48029034 48367254 48447649 48870458 50257119 51876896 51915315 52954495 53239979 53382814 53808023 53914735 54181961 55624222 56091608 56166839 56810401 57403650 58866934 59263875 59905277 59984287 61920784 62967818 63428947 63844595 64420087 65391680 65546401 69231393 69467429 70283553 71115728 71802827 72619726 72746716 73978537 74385744 75036087 76112589 76319237 77023262 78344727 78733844 78979940 80182788 80295395 80401124 81701495 81905323 82314848 82742806 83309656 84202717 84730389 87753367 88002707 88108201 88173537 88982057 89121983 89204509 89790742 90339543 90509104 91687346 91820816 92791781 93363413 93791170 94328524 94498378 95849823 95967328 96211790 96780417 96968123 97425455 97455499 97656876 97741133 97826756 99445333 99451253 100131398 100775017 101124473 101841566 101912546 102267079 102809574 103675406 103880681 104858025 105611584 106323071 106519005 106804424 107620954 108156991 108704132 109745099 109851953 110723856 111038172 111427050 112387691 112929899 113620018 113766592 114362835 115831909 116392583 119342975 121714013 121897578 122159304 123371486 123672906 125663349 128194789 129031326 130320566 131889922 133251320 133931363 134549551 135936111 136510654 136516152 136687766 136881450 136914670 138132001 138698475 139596467 139734966 139985303 140732326 141632865 142516543 143299804 143839914 143970237 144355738 144550442 146042696 146117963 147705893 147787007 148017947 150342043 150516310 151001728 152074946 152564698 153777173 154369170 154421609 154591024 154810640 156126603 156154954 156299977 156661001 156776338 157000018 158360386 159685054 159783503 159822809 160327580 160381522 160943677 163337498 165280926 166042680 166244525 167143774 167346293 169695623 169898727 171112731 171175730 171617177 172210072 172965475 173119599 173967340 174358261 174932508 175413509 176327326 176578960 177173679 177241652 177917534 178244080 178776180 179901938 180605354 180986541 181160469 181219573 181600990 181883472 182123692 183558956 183652206 183830569 184135288 184870490 184947054 185109867 186141137 186379212 187592073 187886901 188136557 188207065 188861574 188890671 189356753 190337113 190658094 190696771 192162728 192547704 192957182 193239684 194977463 195120641 196396399 196627913 197303402 197325472 198983515 199416588 200325702 200620321 200760402 201271958 201313216 201839775 202034736 202833443 203307311 204706079 204890753 205424771 205579279 205636335 206327812 209175112 209996776 210046096 210132081 210779113 210854618 210913348 211138333 211205204 212394610 213029938 213199132 213465539 213864515 215859011 215895740 217834459 219383846 219418016 219524326 220155586 220323606 220545466 221575983 222204717 222668728 222981564 223220716 223517478 223606560 223907367 223985345 224012996 224499499 224864431 225134938 226411123 226657577 227230973 227249542 228451302 229736391 230572013 230676938 230779146 231352847 231514262 231934228 232108964 233158913 233347025 233944883 234031897 235057713 235114705 235510466 236246764 236566483 236732488 237529207 237549229 238189898 239185043 239968230 240213485 240436378 240450574 240854404 242265703 243159333 243174527 243370949 243666625 243912932 244347632 244831515 245360700 246401855 246600128 247013058 248068450 248126868 249615256 249969109 249994738 251479040 252507710 253109165 253133444 254809256 254988167 255263665 257807149 258150574 258705994 259142294 259181044 259223069 260529614 261827183 262359293 264307139 264699164 264794127 265043180 265822235 266191449 266670519 267893484 268636245 271807320 272346302 272521449 274572886 274706756 275439768 275828511 276185375 276331974 276866842 277336154 277399057 277927784 277987282 279045500 280128314 280880294 281826794 282407231 284535407 285322380 285399786 287132613 287768193 287896095 288055103 288291194 288317358 288426023 290358779 291624270 293921992 294283614 295079085 295101014 296122737 297822342 300175569 302345432 305910195 306917042 307456222 309567080 309997025 310268189 311443276 311778148 312102292 312240491 312663342 312978078 313056987 313912829 313914392 314359591 314482372 315595689 315762088 316149440 316219502 316712976 317440955 317744331 317857373 318176289 318500689 320942275 321001788 321697661 323336407 323952051 327607545 329144120 329720520 330401036 331200961 331280192 331523733 331894027 334372336 335061494 336260651 337406478 337968945 338083129 340052766 340156458 340360583 340756996 341112681 341915461 341931822 342368889 343682723 345242936 345296828 345812704 346145166 347473661 347517689 348649383 348882146 349468217 349527602 350380021 352601271 352825744 353005071 353398105 353522988 353966458 355431962 357597205 357692671 358322838 359012870 359237476 359851107 360302303 361208253 361716142 362984279 363070671 363205758 363606735 364278735 364445473 364554866 364786148 365500967 366509566 366552917 367516855 370809545 370900639 371321017 371660164 371871857 372160462 372194416 372657734 373468789 373645173 374226015 374616898 374722711 375515804 375975341 376934762 377171525 377953844 378435263 379260496 379555851 379569309 379807800 380145384 382969075 383765959 383863993 384012315 384305975 384687364 385171937 385330116 385399567 386241569 386449135 386951187 387163846 387296562 387829608 388073343 388273172 389425591 389643592 389796409 390847672 391561437 391971619 391982050 393154182 393664343 394022230 394098732 394741879 395139848 395227637 395902617 396903416 397327165 397391961 397833848 397863137 397888299 398492597 398605914 399636297 400055858 400136654 400637066 401286354 402324419 402698405 402829074 403714100 404205785 404720563 406239259 406265376 406337377 406528488 406966919 407599344 407962904 407968682 408250681 408843199 409684768 413632695 414642848 414827250 414952909 415270095 415474100 416338251 416488470 418362696 418542451 418829334 419203249 420737985 420894654 420922520 421521920 421746483 422486515 422914162 423629032 423663137 423752887 425148679 425583864 427207684 427548422 427776758 427943443 428700315 431614954 432781851 434481017 434963272 435336215 435870242 436397529 436878883 436907112 438445802 439388867 439748680 440976263 441344846 441398208 441717766 441726132 443992194 444104920 444336703 444470933 444622564 444631990 444718643 444832950 445092748 445758001 446368615 447615693 448092051 448516736 449127051 449250213 449558599 449572845 450474411 451176773 451845503 452223155 452447416 452653153 452995219 453976288 454602786 455486670 458418983 462907229 463934297 464042030 464667473 464708445 465258744 466811940 466932365 468323121 468986974 470035822 470099524 470514310 470792237 471092270 472175645 472794109 473537428 473835395 474840554 475499998 476181366 476588396 477153303 477351316 478449666 479152191 479473770 481843083 482331836 482581110 482718552 482776289 483940249 484622901 485664851 485939356 487029916 487668167 489872834 489878282 490275621 491294326 491369720 491872175 492974053 494039901 494336669 494441720 494823330 496334943 497134841 498701313 498910819 501167766 501449679 501780999 503213227 504216284 504525242 505312363 505679056 506508371 506554806 507300534 507378936 508710325 508988073 511817277 511954636 512684271 513319543 515797509 516179571 518094309 519730970 520613243 520669028 521224702 521429655 522019452 523631568 523976231 524080644 524617597 525860831 525994305 526248353 528049031 529465140 529498769 529855445 530778955 530958482 531275934 532971889 534829089 534909220 535916206 537175419 538484291 539518261 541247929 541953194 542348197 542510708 543334368 544778593 544877767 545036177 545132876 545175031 546336979 546507658 546879177 546885175 547318620 547570749 548513063 549087573 550142089 550286411 550429683 550966373 551886905 552782864 554217679 554742986 555001506 555777935 558718975 558750939 560617722 561616498 562249718 563160993 563508376 563790614 564875448 565035158 565655374 566754369 570184182 571077767 571795496 571941157 571956148 572041847 572129672 572298445 573165441 574180514 574285032 575480953 576096703 576305284 576510306 578468927 578729377 579556215 581086956 581130587 581340002 582412826 583898080 584264689 584322540 584678656 585692031 585900178 585931230 588136486 589424485 590969835 592498969 593547584 594064211 595595467 596776439 599890627 600434641 600450308 600933618 602215383 602266161 603581372 603880642 605202611 605721356 606272982 606316879 606466778 608270691 609384409 609824701 612058945 612377575 612585912 612714921 614065766 615354323 616104263 616808557 619257152 619541396 620171242 620302290 620358373 620529366 621221549 621232969 621423530 622178934 622247430 623080984 624004617 624710123 625162692 627747663 628123361 628627330 629446916 629999798 631481870 632549153 632905467 634972676 635678800 635902318 636044044 637693872 638432853 638869143 639393217 640212290 640641618 640800231 641581096 641653753 641733567 643010718 643623212 643923585 646625510 646715912 649405416 649433158 650305489 650484107 651119777 652017532 652586757 652869334 654083425 656060150 658468644 658652563 659025604 659389603 660007389 663747529 663791862 664825706 664950517 665003266 665851284 666320112 666756709 666814698 666880059 667238174 667694526 668794513 668853641 669498901 670664654 671377508 671669538 673564513 674109524 674905295 675198430 675629942 675857380 677548358 677694702 678495820 678588487 680817329 681042329 682156516 682751534 683685276 684432713 684792495 685126631 686355890 687053269 687641029 689424797 690254535 691976172 693627218 694774214 695820551 695977145 697406660 697480534 697737799 698938787 699142938 699503282 700004861 701230492 701667402 702123728 702585616 710054202 710231733 710278459 710981403 712791593 712992286 714484721 716253748 717218595 718098437 718595206 720221287 720390862 721623262 721972335 722192910 722940002 723295925 725579629 726430971 726531266 727280796 727533789 728252480 729819579 729827889 730693753 730979645 731436340 732111365 732306490 732328669 733355604 733437432 733853349 734367520 735866698 736090998 736955266 737979006 738368448 738715121 741304272 741630896 742471367 742805320 743797150 744219407 744226887 744502999 744721770 745337652 745839039 746077647 746950558 747072086 747294392 747426020 748131961 748565907 748650359 749147593 749207729 749693964 752566210 752853270 754565203 754607741 754796571 755466231 755861696 756050850 757300220 757420641 758245580 760231924 761989647 762055457 762745560 763223651 763366544 763852680 763914629 763979206 764617712 765460507 765679480 765768062 765787727 767456351 768241636 769206347 769221170 770318613 770920064 771215243 771322580 771894632 772023281 772047411 772285046 773880828 774199208 774929909 775877572 777393744 777451286 779599446 779830930 780219504 781623845 782150438 782423764 782534257 782535437 783398898 784019363 786528964 788977337 789183182 789373358 789521309 789774831 790434932 791022789 791876257 791999192 792353743 794122343 794671203 797658292 798238331 798625362 800912801 801492065 801634037 801793365 802833076 803134882 803261604 803708024 803712915 804043384 804078428 804686952 804724618 807334688 808668087 809017189 810332744 811353781 811788668 812152137 812184660 813668440 814099788 814735553 815465069 817271490 818201263 818457228 818530859 818898211 819430309 819653889 820532129 821681232 822288123 822341609 822465677 823895558 825278544 825750954 825950789 826027858 826473388 826766799 826934879 828847353 829197887 829723867 830560104 831675041 831927945 832959991 833054794 833215786 833668109 834479898 836160958 837288157 837431003 838139365 838939064 838941100 838966098 840975752 841559677 841916563 842074093 843075397 843298610 845694904 846118302 847247166 847251105 848214639 848547691 849371559 849690780 851600767 851787516 852287592 852463159 853286379 853447260 855396965 856260757 856559058 856598441 857706340 858843145 859257055 859397658 862361212 862450027 862602214 863963196 864711799 864853689 864902727 864980380 867043946 867057535 867587662 868271218 868391016 868772916 868911187 869815376 870317761 870679304 870789564 870886243 871358283 872032981 872941795 874288122 874439630 875351637 875649170 876278612 877778163 878397192 879478751 882669432 882716561 883104107 884575004 885490654 885534410 885901186 886120087 887221897 887961080 888678715 888872967 890714002 891386553 892168269 892302162 893159845 893231585 893273814 893638077 894544888 894598943 895171623 895351905 895955233 898047723 899427546 899550864 899611902 900192067 900395243 901636022 902093427 902145357 903010321 903983064 904060899 904821592 905820834 906896024 907752388 907949999 908165894 908260932 908879928 909580712 909586660 909892755 910327233 913504025 914245151 915492492 919934800 920304088 920976069 922469021 923063431 923274817 925098711 926163592 926802712 927115886 927273293 927454479 927661979 928139648 928771381 928868261 929925966 930001898 930797608 933417512 933425602 934425479 935655019 935759129 936422637 936998235 937931898 937976656 939079486 939238881 942733648 943629824 944152127 944615564 944812795 944885305 945049022 945178496 946780836 947373712 949257177 949286976 949720462 950261938 950446752 950569436 950620390 950869117 951650538 952030773 952048078 952075746 952584529 953430924 953734140 955263962 955335056 955336217 956405616 956465292 956521127 956769508 956885140 957218955 957278669 957419562 958942317 959330646 959728515 959832849 961238428 961873780 962134828 962276775 963689806 964369661 964538307 966003783 967154442 967303032 967415648 967528169 968032663 968111513 968814578 969010338 970067203 971211638 971247196 971365911 971973047 972159676 974605526 974619034 974639285 974761303 974824399 975575123 976251348 976797979 978130324 978184339 979496112 982157374 982426407 982911870 983177942 983225441 983921903 984047050 985166447 986220330 986547569 988005700 988921038 990020371 990123986 990528527 990733911 991967791 993315821 993609792 993981455 994308718 994470020 996014833 996261701 997446293 998051277 998956656 999108953 999286112 999627319
