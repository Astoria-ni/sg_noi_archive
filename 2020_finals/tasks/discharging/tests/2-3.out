1428922571601
